CODEOWNERS

```
@@ -3,3 +3,4 @@ check-links.sh  @pcolmer @morancj
 CODEOWNERS      @pcolmer @morancj
 Gemfile         @pcolmer @morancj
 manifest.json   @pcolmer @morancj
+_data/routingrules.json  @pcolmer @morancj
```

Do we want to move to using teams instead of individuals for `CODEOWNERS`? e.g:
```suggestion
_data/routingrules.json  @orgs/linaro-its
```
See [About code owners#CODEOWNERS syntax](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax)